app/bundles/CoreBundle/Tests/_support/FunctionalTester.php

```diff
@@ -1,28 +0,0 @@
-<?php
-
-namespace Mautic\CoreBundle;
-
-/**
- * Inherited Methods.
- *
- * @method void                    wantToTest($text)
- * @method void                    wantTo($text)
- * @method void                    execute($callable)
- * @method void                    expectTo($prediction)
- * @method void                    expect($prediction)
- * @method void                    amGoingTo($argumentation)
- * @method void                    am($role)
- * @method void                    lookForwardTo($achieveValue)
- * @method void                    comment($description)
- * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
- *
- * @SuppressWarnings(PHPMD)
- */
-class FunctionalTester extends \Codeception\Actor
```

Comment:
I was looking at where this file come from. It looks like it's a start to support Codeception tests. See the readme changes in https://github.com/mautic/mautic/pull/4868

It looks like the codeception library was since removed from composer.json so it's not useable. I'm wondering whether we shouldn't remove the rest. Like the functional.suite.yml file.

Reply:
it's not really related to phpstan, but it's cleanup :) So I added that in https://github.com/mautic/mautic/pull/12225/commits/27bca7df749bf87f4bb78c57c2220793c8d37ab7